openpype/hosts/max/plugins/load/load_pointcache.py

```diff
@@ -52,14 +54,47 @@ def load(self, context, name=None, namespace=None, data=None):
 
         abc_container = abc_containers.pop()
 
-        container_name = f"{name}_CON"
-        container = rt.container(name=container_name)
-        abc_container.Parent = container
+        return containerise(
+            name, [abc_container], context, loader=self.__class__.__name__)
 
-        return container
+    def update(self, container, representation):
+        from pymxs import runtime as rt
+
+        path = get_representation_path(representation)
+        node = rt.getNodeByName(container["instance_node"])
+
+        alembic_objects = self.get_container_children(node, "AlembicObject")
+        for alembic_object in alembic_objects:
+            alembic_object.source = path
+
+        lib.imprint(container["instance_node"], {
+            "representation": str(representation["_id"])
+        })
+
+    def switch(self, container, representation):
+        self.update(container, representation)
 
     def remove(self, container):
         from pymxs import runtime as rt
 
         node = container["node"]
         rt.delete(node)
+
+    @staticmethod
+    def get_container_children(parent, type_name):
+        from pymxs import runtime as rt
+
+        def list_children(node):
+            children = []
+            for c in node.Children:
+                children.append(c)
+                children = children + list_children(c)
```

Comment:
```suggestion
                children += list_children(c)
```
Or
```suggestion
                children.extend(list_children(c))
```

Note that this could technically hit a recursion limit if a scene were ever nested deeper than 1000 levels.